### ISSUE TITLE
Image can't be pulled

### ISSUE THREAD

**Issue by BernhardFuchs** [caict-4iot-dev/uni-resolver-driver-did-bid#1]
When pulling the image with `docker pull caictdevelop/driver-did-bid:latest` it can't be found.  

According to our [rules](https://github.com/decentralized-identity/universal-resolver/blob/main/docs/driver-development.md#how-we-handle-problems-with-drivers) we are removing drivers which have no image available. The reason for this is that they are consuming resources by trying to pull the image constantly. See this [commit](https://github.com/decentralized-identity/universal-resolver/commit/fdd6dae8aaeaab24b804fddf124ba5b464fb630c). 

Once your issues are solved, feel free to readd the driver.

**Comment by peacekeeper** [comment]
Yes it looks like there are no images here: https://hub.docker.com/r/caictdevelop/driver-did-bid

**Comment by Caict-GuoShiJie** [comment]
The mirror has been re-released. Please retry. Thank you for your patience.

**Comment by peacekeeper** [comment]
Looks like it's working again: https://dev.uniresolver.io/#did:bid:ef214PmkhKndUcArDQPgD5J4fFVwqJFPt

Thanks for fixing, I think this issue can be closed.
